wof/core.py

```diff
@@ -49,7 +49,8 @@
 except ImportError:
     version = 'dev'
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
+_root_dir = os.path.abspath(os.path.dirname(__file__))
+_TEMPLATES = os.path.join(_root_dir, 'app', 'templates')
```

Comment:
@ocefpaf okay. I will merge this PR since it appears to only change the paths to templates folder, using relative paths, instead of absolute, which does help windows user. I will modify my documentation to use the same pattern. Thanks!